### ISSUE TITLE
Display Glitches on Mouse Movement

### ISSUE THREAD

**Issue by Hubok** [richardwilkes/gcs#197]
![](https://i.imgur.com/2rn6RLj.png)
Version: 4.16.0, Windows 10 (normal MSI installer)

This would frequently glitch out and clear up when moving over any of the tabs, interacting with the library explorer, or using the function menu. Started after opening up the Preferences menu. After restarting GCS I was unable to reproduce this, however I'll try to find a reliable way to figure out what caused it.

**Comment by richardwilkes** [comment]
Exceptions are being thrown. It could be something on the system, such as low video RAM causing problems, or it could be a bug in GCS. Try enabling logging to a file since Windows throws away console output by default. You'll need to set the `GCS_LOG` environment variable to a path to a file to write the log to, then launch GCS. If you see this again, there will hopefully be some output in that file you can attach to the bug report.

**Comment by Hubok** [comment]
Just summarizing our quick chat from Discord. VRAM is not an issue, but I do have SLI enabled, so this is my fault. I've enabled logging and I'll keep an eye out - I managed to get the glitch for the second time, it's only happened on 4.16.0 for me (4.15.1 and previous never did it), and it only seems to happen if I leave GCS open for a reasonably long amount of time.

I've attached my GPU topology for reference.
![](https://i.imgur.com/mnsOXWI.png)
